packages/ozone/tests/__snapshots__/get-record.test.ts.snap

```diff
@@ -39,6 +39,9 @@ Object {
       },
       "subjectBlobCids": Array [],
       "subjectRepoHandle": "alice.test",
+      "tags": Array [
+        "lang:unknown",
```

Comment:
I assume we like the `lang:unknown` as it's explicit, versus not having a lang tag?

Reply:
yeah and also, this helps reduce unnecessary calls to figure out `lang` tag on every event.
since this tagger runs on every event for a subject unless the subject already has a `lang` tag, failing to figure out the right tag once probably means future attempts will fail too so we will just avoid running it again.

Reply:
the 3-char code `und` ("Undetermined") is a pseudo-specified way to represent this.

thinking ahead to UI stuff... we may want to have a special/specific drop-down/conversion for language tags, different from other tags? But I think we can get to that whenever, doesn't need to be first iteration.